src/latexify/ipython_wrappers.py

```diff
@@ -0,0 +1,135 @@
+"""Wrapper objects for IPython to display output."""
+
+from __future__ import annotations
+
+import abc
+from typing import Any, Callable
+
+from latexify import exceptions, generate_latex
+
+
+class LatexifiedRepr(metaclass=abc.ABCMeta):
```

Comment:
Add type hints for every function.

Reply:
`_repr_something_` should return `str | tuple[str, dict[str, Any]] | None` by design, where the second case involves [metadata](https://ipython.readthedocs.io/en/stable/config/integrating.html#metadata)